smarts/core/utils/file.py

```diff
@@ -74,3 +74,11 @@ def path2hash(file_path: str):
     m = hashlib.md5()
     m.update(bytes(file_path, "utf-8"))
     return m.hexdigest()
+
+
+def file_md5_hash(file_path: str) -> str:
```

Comment:
lrucache this? (learned that trick from @Gamenot)

Reply:
I thought about this but I don't think caching is helping because scenario folders are not duplicated in most cases.